### DISCUSSION TITLE
Gaszaehler rate to high

### DISCUSSION THREAD

**Discussion by alero63** [jomjol/AI-on-the-edge-device#2643] [General] [answered]
Hi, habe mir auch den Gaszaehler mit einer ESp32 Cam ausgestattet. leider bekomme ich immer rate to high angezeigt.
FM:
`Rate too high - Read: 2147483647 - Pre: 2147483647 - Rate: 2147483647`
Previous Value gesetzt auf 6389

Hier meine Konfig.
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/46607155/69d2236e-64cf-4fae-90d6-efb14bdf2eab)

**Comment by tomquist** [comment, marked as answer]
Decimal shift sollte vermutlich eher auf -3 statt 3 gesetzt werden um einen shift nach links zu erzielen. Bei meinem Gaszähler habe ich Maximum Rate Value außerdem auf 1 gesetzt.

**Reply by alero63** [reply]
Danke das wars. Hab aber auch noch was bei meinen gaszaehler.

**Comment by alero63** [comment]
@tomquist hi, danke werde ich ausprobieren.

habe auch noch einen Fehler beim wasserzaehler. Vielleicht hast du ja auch eine Idee.
FM:
`Neg. Rate - Read: - Raw: 992.249215 - Pre: 2249932.000000`
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/46607155/cc17f348-c6fb-499f-8141-bf7dfe469916)


![image](https://github.com/jomjol/AI-on-the-edge-device/assets/46607155/904473e9-1c30-495c-a56d-ad4eb53d17a7)

**Reply by tomquist** [reply]
Habe selbst leider keine Erfahrung mit analogen ROIs. Am Besten mal eine neue Frage posten.
